src/Swarm/Game/Scenario/Style.hs

```diff
@@ -0,0 +1,70 @@
+module Swarm.Game.Scenario.Style where
+
+import Brick (AttrName, attrName)
+import Data.Aeson
+import Data.Set (Set, toList)
+import Data.Text (Text)
+import Data.Text qualified as T
+import GHC.Generics (Generic)
+import Graphics.Vty.Attributes
+import Numeric (readHex)
+import Swarm.TUI.Attr (worldPrefix)
```

Comment:
I agree with @xsebek , I think it would be better to do this in a way that didn't depend on anything in `Swarm.TUI.*`, partly just out of a sense that conceptually the abstract game should not depend on the specifics of the user interface, and partly for the practical reason that we might want to have different interfaces in the future.